recipes/mac-and-cheese.qmd

```diff
@@ -0,0 +1,94 @@
+3.  While water is coming up to a boil, shred cheeses and toss together to mix, then divide into three piles. Approximately 3 cups for the sauce, 1 1/2 cups for the inner layer, and 1 1/2 cups for the topping.
+4.  Melt butter in a large saucepan over MED heat. Sprinkle in flour and whisk to combine. Mixture will look like very wet sand. Cook for approximately 1 minute, whisking often. Slowly pour in about 2 cups or so of the half and half, while whisking constantly, until smooth. Slowly pour in the remaining half and half and the whole milk, while whisking constantly, until combined and smooth.
+5.  Continue to heat over MED heat, whisking very often, until thickened to a very thick consistency. It should almost be the consistency of a semi thinned out condensed soup.
+6.  Remove from the heat and stir in spices and 1 1/2 cups of the cheeses, stirring to melt and combine. Stir in another 1 1/2 cups of cheese, and stir until completely melted and smooth. Stir in the jalapeño.
+7.  In a large mixing bowl, combine drained pasta with cheese sauce, stirring to combine fully. Pour half of the pasta mixture into the prepared baking dish. Top with 1 1/2 cups of shredded cheeses, then top that with the remaining pasta mixture.
+8.  Sprinkle the top with the last 1 1/2 cups of cheese and bake for 15 minutes, until cheesy is bubbly and lightly golden brown.
+9.  While baking, toast some panko breadcrumbs on the stovetop over medium heat to top the final dish with.
+
+### Tips
+
+-   Please feel free to add additional seasonings per your tastes, this is a base recipe that you can change up to make your own. Garlic powder, onion powder, ground mustard, smoked paprika, and hot sauce are some favorites of ours to add.
+-   To get the 4 cups of cheddar, use a 16 oz block of cheddar and shred.
+-   To get the 2 cups of gruyere, use as close to an 8 oz block of gruyere as you can (different brands have different sized blocks), and shred.
+
+#### Mac and Cheese can be made ahead:
+
+-   Make as directed (but do not bake), transfer to baking dish and cool completely. Cover tightly with foil and refrigerate 1-2 days ahead.\
+-   Before baking, let dish sit on counter for 30 minutes.
+-   Bake at recipe temperature for 25-35 minutes, until hot and bubbly.
+
+#### Five cheese Variation:
+
+Replace cheese amounts with the following: \* 2 cups cheddar cheese, shredded \* 1 cup Colby cheese, shredded \* 1 cup Muenster cheese, shredded \* 1 1/2 cups Gruyere cheese, shredded \* 1/2 cup mozzarella cheese, shredded
+
+#### For Crunchier Topping and Firmer Consistency:
+
+Prepare recipe as directed but bake at 350 or 375 F degrees for 20-30 minutes. Broil at the end of baking for approximately 2-5 minutes, until golden brown and crunchy.
+
+Combine with tomatosoup for a perfect meal:
```

Comment:
It seems like a new recipe starts here? @cdollst could you please remove this? Thanks!